gnd/src/androidTest/java/com/google/android/gnd/CustomTestRunner.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd;
+
+import android.app.Application;
+import android.content.Context;
+import androidx.test.runner.AndroidJUnitRunner;
+import dagger.hilt.android.testing.HiltTestApplication;
+import timber.log.Timber;
+import timber.log.Timber.DebugTree;
+
+public class CustomTestRunner extends AndroidJUnitRunner {
+
+  @Override
+  public Application newApplication(ClassLoader cl, String className,
+    Context context)
```

Comment:
Was this kind of indentation allowed in Java? D:

Reply:
Nope :) All fixed now.